composer.json

```diff
@@ -34,6 +34,7 @@
 	},
 	"require-dev": {
 		"leafo/lessphp": "~0.4.0",
+		"leafo/scssphp": "^0.3.0",
```

Comment:
I find "~0.3" a bit more expressive than this.  
This gives off the notion of a very specific version, but actually, it's ">= 0.3.0 <1.0.0" (at least since composer 1.0.0 ) https://getcomposer.org/doc/articles/versions.md#caret